### DISCUSSION TITLE
Section 1: "volunteer-driven ecosystem"

### DISCUSSION THREAD

**Discussion by hamogu** [astropy/astropy-v5.0-paper#66] [Text comments] [answered]
> has been the significant, volunteer-driven effort behind developing community-
oriented [tools]

I'm not sure that's correct. For example, in the list of packages that follow, there is tensorflow, developed by Google. matplotlib at large contributions from STScI, numpy as well, I don't know much about the history of the others. How about " One of the factors ... is the combination of volunteer-driven and professional/commercial software development ..."

**Comment by pllim** [comment]
Addressing this probably the same effort as addressing #67 ?

**Comment by adrn** [comment, marked as answer]
I addressed this in a commit that I will push soon. Thanks!
